src/maker.rs

```diff
@@ -279,24 +349,21 @@ mod tests {
         let event = maker.process_taken_order(order_taken).unwrap();
 
         assert_eq!(event, TakeRequestDecision::GoForSwap);
-        assert_eq!(
-            maker.btc_reserved_funds,
-            bitcoin::Amount::from_btc(1.5).unwrap()
-        )
+        assert_eq!(maker.btc_reserved_funds, btc(1.5))
     }
 
     #[test]
     fn btc_funds_reserved_upon_taking_sell_order_with_fee() {
         let mut maker = Maker {
-            btc_balance: bitcoin::Amount::from_btc(3.0).unwrap(),
-            btc_fee: bitcoin::Amount::from_btc(1.0).unwrap(),
+            btc_balance: some_btc(3.0),
```

Comment:
```suggestion
            btc_balance: Some(btc(3.0)),
```

I am not sure these helpers functions are that helpful, you save 1 character per call.

Reply:
It's just faster when typing because I only have to auto-complete one function call :D 
If you feel strong about that one I'd suggest a follow up.